apps/generic_terminal/generic_unix_shell.py

```diff
@@ -1,11 +1,39 @@
+from pathlib import Path
+
 from talon import Context, Module, actions
 
+from ...code.user_settings import get_list_from_csv
+
 ctx = Context()
 mod = Module()
 ctx.matches = r"""
 tag: user.generic_unix_shell
 """
 
+path = Path(__file__).parents[0]
+with open(path / "default_unix_utilities.csv") as f:
+    # get_list_from_csv() expects the columns in the reverse order
+    default_unix_utilities = {
+        spoken.strip(): command.strip()
+        for command, spoken in [line.split(",", 1) for line in f]
+    }
+
+unix_utilities = get_list_from_csv(
+    "unix_utilities.csv",
+    headers=("command", "spoken"),
+    default=default_unix_utilities,
+)
+
+mod.list("unix_utility", desc="A common utility command")
+ctx = Context()
+ctx.lists["self.unix_utility"] = unix_utilities
+
+
+@mod.action_class
+class UnixUtilityActions:
+    def terminal_run_unix_utility(command: str):
+        """Runs a utility command"""
+
```

Comment:
This declares the action `terminal_run_unix_utility` but doesn't implement it. Did you forget to add that to the PR?

Reply:
Seems that way. Anyways, as I removed the action as per your suggestion, this is not relevant anymore. I'm going to mark this specific issue as resolved.